packages/node-core/src/indexer/store.service.ts

```diff
@@ -267,23 +269,51 @@ export class StoreService {
   }
 
   async getHistoricalStateEnabled(): Promise<boolean> {
-    let enabled = true;
+    const {disableHistorical} = this.config;
+    const multiChainHistoricalError =
+      'Cannot multi-chain index when historical state is enabled, try using --disable-historical';
```

Comment:
I think we want to change this behaviour. 

If historical is enabled when the project is started theres no way to disable it. I think we should keep that. So if a user tries to change to multi chain and theres any historical data we can throw an error and tell them to index from scratch